src/jit/inlinepolicy.cpp

```diff
@@ -26,7 +26,19 @@
 
 InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
 {
-    // For now, always create a Legacy policy.
+
+#ifdef DEBUG
+
+    // If JitInlineRandomSeed is set, use the RandomPolicy.
+    unsigned seed = JitConfig.JitInlineRandomSeed();
```

Comment:
Interesting. But (as your probably well know) `stressLevel` only influences which stress modes are enabled, not how a particular stress mode behaves when it's enabled.

I suppose I can fold random inlining into the existing jit stress. My main motivation is to assess how likely alternative inlining strategies will uncover issues in the jit, so I'm happy so long as I can run with just random inlining enabled and can also vary the inlining patterns. Tying it into stress and providing an additional seed seems like it would work out ok.

I'm still not sure what the code in `getPolicy` would look like in this case... will work on it a bit.